docs/data/date-pickers/base-concepts/ComponentFamilies.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import { DemoContainer, DemoItem } from '@mui/x-date-pickers/internals/demo';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateField } from '@mui/x-date-pickers/DateField';
+import { TimeField } from '@mui/x-date-pickers/TimeField';
+import { DateTimeField } from '@mui/x-date-pickers/DateTimeField';
+import { MultiInputDateRangeField } from '@mui/x-date-pickers-pro/MultiInputDateRangeField';
+import { MultiInputTimeRangeField } from '@mui/x-date-pickers-pro/MultiInputTimeRangeField';
+import { MultiInputDateTimeRangeField } from '@mui/x-date-pickers-pro/MultiInputDateTimeRangeField';
+import Stack from '@mui/material/Stack';
+import Tooltip from '@mui/material/Tooltip';
+
+function ProLabel({ children }: { children: React.ReactNode }) {
+  return (
+    <Stack direction="row" spacing={0.5}>
+      <Tooltip title="Included on Pro package">
+        <a href="/x/introduction/licensing/#pro-plan">
+          <span className="plan-pro" />
+        </a>
+      </Tooltip>
+      <span>{children}</span>
```

Comment:
Maybe `Included in` is semantically a bit more correct? 🤔 
Also, inverting order of icon and text seems a bit more correct based on other usages.
```suggestion
      <span>{children}</span>
      <Tooltip title="Included in Pro package">
        <a href="/x/introduction/licensing/#pro-plan">
          <span className="plan-pro" />
        </a>
      </Tooltip>
```